proto/README.md

```diff
@@ -63,3 +70,4 @@ positions | meters | Vector3 (list) | List of object’s XYZ position.
 Field | Unit | Type | Description
 --- | --- | --- | ---
 positions | meters | Vector3 (list) | List of object’s XYZ position.
+reachable_set | meters | [ReachableSet](sensr_proto/type.proto) | List of object’s reachable set.
```

Comment:
how about `List of the predicted range which includes uncertainty sorted by time-lapse.`?

Reply:
Can I replace `the predicted range` -> `object's predicted range`?

Reply:
yup sure.